x-pack/plugins/security_solution/server/lib/detection_engine/migrations/create_migration.ts

```diff
@@ -59,6 +59,23 @@ export const createMigration = async ({
                   ctx._source.signal._meta = [:];
                 }
                 ctx._source.signal._meta.version = params.version;
+
+                // migrate enrichments before 7.15 to ECS 1.11
+                if (ctx._source.signal?.rule?.type == "threat_match" &&
+                ctx._source.threat?.indicator instanceof List &&
+                ctx._source.threat?.enrichments == null) {
+                  ctx._source.threat.enrichments = [];
+                  for (indicator in ctx._source.threat.indicator) {
+                    def enrichment = [:];
+                    enrichment.indicator = indicator;
+                    enrichment.indicator.reference = indicator.event?.reference;
+                    enrichment.matched = indicator.matched;
+                    enrichment.indicator.remove("matched");
```

Comment:
Should `enrichment.indicator.event` be removed as well?

Reply:
Nvm, just saw the comment above explaining the logic

Reply:
I opted to leave it in case existing CTI users want/need that info (albeit unmapped).